nixos/modules/services/monitoring/scrutiny-collector.nix

```
@@ -0,0 +1,141 @@
+      default = "http://localhost:8080";
+      description = mdDoc ''
+        The api server endpoint
+      '';
+    };
+
+    log-file = mkOption {
+      type = types.nullOr types.path;
+      default = null;
+      description = mdDoc ''
+        Path to file for logging. Leave empty to use STDOUT
+      '';
+    };
+
+    debug = mkEnableOption "debug logging";
+
+    host-id = mkOption {
+      type = types.str;
+      default = config.networking.hostName;
+      defaultText = lib.literalExpression "config.networking.hostName";
+      description = mdDoc ''
+        Host identifier/label, used for grouping devices
+      '';
+    };
+
+    calendar = mkOption {
+      type = types.str;
+      default = "*-*-* 00:00:00";
+      description = mdDoc ''
+        Configured when to run the service systemd unit (DayOfWeek Year-Month-Day Hour:Minute:Second).
+      '';
+    };
+
+    user = mkOption {
+      type = types.str;
+      default = "scrutiny-collector";
+      description = mdDoc ''
+        User under which the scrutiny collector service runs.
+      '';
+    };
+
+    group = mkOption {
+      type = types.str;
+      default = "scrutiny-collector";
+      description = mdDoc ''
+        Group under which the scrutiny collector service runs.
+      '';
+    };
+  };
+
+  config = mkIf cfg.enable
+    (
+      let
+        configFile =
+          if cfg.configuration != null then
+            (pkgs.writeTextFile {
+              name = "collector.yaml";
+              text = (lib.generators.toYAML { } cfg.configuration);
+            })
+          else null;
+        createUser = cfg.user == "scrutiny-collector";
+      in
+      {
+        environment.systemPackages = [ cfg.package ];
+
+        users.groups.${cfg.group} = lib.mkIf createUser { };
+        users.users.${cfg.user} = {
+          description = lib.mkIf createUser "Scrutiny Collector Service User";
+          createHome = lib.mkIf createUser false;
+          isSystemUser = lib.mkIf createUser true;
+          group = lib.mkIf createUser cfg.group;
+          extraGroups = [ "disk" ];
+        };
+
+        systemd = {
+          timers.scrutiny-collector = {
+            description = "Scrutiny Collector";
+            wantedBy = [ "timers.target" ];
+
+            timerConfig = {
+              OnCalendar = cfg.calendar;
+              AccuracySec = "5m";
+              Unit = "scrutiny-collector.service";
+            };
+          };
+
+          services.scrutiny-collector = {
+            description = "Scrutiny Collector";
+
+            serviceConfig =
+              let
+                cmd = "${cfg.package}/bin/collector-metrics run";
+                args = lib.cli.toGNUCommandLineShell { } {
+                  inherit (cfg) api-endpoint log-file debug host-id;
+                  config = if cfg.config-path != null then cfg.config-path else configFile;
+                };
+              in
+              {
+                ExecStart = "${cmd} ${args}";
```

is `cmd` changed at all here? can we just do
```suggestion
                ExecStart = "${lib.getExe cfg.package} run ${args}";
```
and remove the def above?